items/src/main/scala/weco/api/items/services/SierraItemUpdater.scala

```diff
@@ -62,6 +72,45 @@ class SierraItemUpdater(sierraService: SierraService)(
           .getOrElse(item)
     } toSeq
 
+  private def getAccessCondition(itemData: SierraItemData): AccessCondition = {
+    // The bib ID is used for debugging purposes; the bib status is only used
+    // for consistency checking. We can use placeholder data here.
```

Comment:
Probably not for fixing here, but it occurs to me that the Sierra API does give us a list of bib IDs in the item responses, and we could just pick one to use here.

Reply:
Oh, that is useful.